src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/FinanceTrackerParser.java

```diff
@@ -172,9 +175,10 @@ public Command parseCommand(String userInput, UiState uiState) throws ParseExcep
         case ConvertBookmarkCommand.COMMAND_ALIAS:
             switch (uiCurrentTab) {
             case EXPENSES:
-                return new ConvertBookmarkExpenseCommandParser().parse(arguments);
+                return new ConvertBookmarkExpenseCommand(
+                        new ConvertBookmarkCommandParser().parse(arguments));
             case INCOME:
-                return new ConvertBookmarkIncomeCommandParser().parse(arguments);
+                return new ConvertBookmarkIncomeCommand(new ConvertBookmarkCommandParser().parse(arguments));
```

Comment:
Should line 178-179 be a single line? Other commands are parsed in a single line too.